functions/routes/ecom/modules/calculate-shipping.js

```diff
@@ -229,9 +229,7 @@ exports.post = ({ appSdk }, req, res) => {
               from: {
                 zip: originZip
               },
-              to: {
-                zip: data.data ? data.data.postalCode : params.to.zip
-              }
+              to: params.to
```

Comment:
Acho que a sua alteração fixaria o erro, mas de qualquer forma parece melhor (para debug e eventuais conflitos) pegar o `postalCode` que o Mandaê retorna como estava antes.
Dessa forma aí pega o CEP deles e mantém o resto do endereço:

```suggestion
              to: {
                ...params.to,
                zip: (data.data && data.data.postalCode) || params.to.zip
              }
```